core/settings.py

```diff
@@ -172,7 +211,19 @@
     "template_partials",
 ]
 
-INSTALLED_APPS = PREREQ_APPS + PROJECT_APPS
+INSTALLED_APPS = (
+    PREREQ_APPS
+    + PROJECT_APPS
+    + [
+        "anymail",
```

Comment:
@jefftriplett do you know why the `anymail` was split into it's own list? I'm good with moving it, just not sure why it would have been explicitly separated and want to make sure we don't make a change before knowing why it is the way it is 😄

Reply:
@ryancheley Nope, probably an oversight or it was shifted during a merge conflict/review.